events/voiceStateUpdate.js

```diff
@@ -92,10 +94,10 @@ module.exports = {
 				if (player.pauseTimeout) {
 					clearTimeout(player.pauseTimeout);
 				}
-				player.pauseTimeout = setTimeout(p => {
+				player.pauseTimeout = setTimeout(async p => {
 					logger.info({ message: `[G ${p.guildId}] Disconnecting (inactivity)`, label: 'Quaver' });
 					p.musicHandler.locale('MUSIC_INACTIVITY');
-					p.musicHandler.disconnect();
+					await p.musicHandler.disconnect();
```

Comment:
there's no need to make this async. timeouts don't work well with async, you can just revert it back